bot/exts/info/information.py

```diff
@@ -42,15 +43,33 @@ def get_channel_type_counts(guild: Guild) -> DefaultDict[str, int]:
         return channel_counter
 
     @staticmethod
-    def get_member_counts(guild: Guild) -> Dict[str, int]:
+    def join_role_stats(role_ids: list[int], name: str, guild: Guild) -> dict[str, int]:
```

Comment:
We should be able to make this useful other than to Leads.
```suggestion
    def join_role_stats(role_ids: list[int], guild: Guild, name: Optional[str] = None) -> dict[str, int]:
```

Reply:
Note: I think we can also come up with a better name